docs/ocean/leap_authorization.rst

```diff
@@ -10,40 +10,92 @@ and higher enable Leap\ |tm| authentication using the secure
 `OAuth 2.0 <https://oauth.net/2/>`_ industry standard. As described in the
 :ref:`ocean_sapi_access_basic` section, you require a Solver API (:term:`SAPI`)
 token from your Leap account to submit problems to Leap
-:term:`solvers <solver>`. The following procedure authorizes the Ocean software
-access to your account in the Leap service and enables storing your SAPI token
-in your development environment. Ocean software authorization to access the Leap
-service persists across subsequent development sessions for this development
-environment.
+:term:`solvers <solver>`.
 
-.. note:: The following `OAuth-based <https://oauth.net/2/>`_ procedure is
-    provided for convenience but you also have the option of manually copying
-    your SAPI token from the dashboard in the Leap service. The
-    :ref:`ocean_sapi_access_basic` section provides more information.
+Authorization Procedure
+=======================
 
-This procedure uses the :ref:`Ocean software CLI <ocean_dwave_cli>` commands
-that you enter into your terminal. You can see help documentation for these
-commands and all their options with the CLI's ``--help`` option.
+The following procedure authorizes the Ocean software access to your account in
+the Leap service and enables storing your SAPI token in your development
+environment. Ocean software authorization to access the Leap service persists
+across subsequent development sessions for this development environment.
+
+This procedure uses the :ref:`Ocean software CLI <ocean_dwave_cli>`
+commands\ [#]_ that you enter into your terminal.
 
 #.  For any new developer environment (for example, a Python
     `virtual environment <https://docs.python.org/3/library/venv.html>`_
     or a `GitHub Codespaces <https://docs.github.com/codespaces>`_ *codespace*),
-    authorize the Ocean software access to your account in the Leap service,
-    using the secure `OAuth 2.0 <https://oauth.net/2/>`_ code exchange, by
-    running one of the following commands from your terminal.
+    run one of the following commands from your terminal. If your account has
+    more than one project, the :ref:`tab_authorize_multiple_leap_projects` tab
```

Comment:
This creates a link to table, and jumps to the tab bar, but it doesn't actually open the second tab.

Reply:
Yes, that's a sphinx limitation, perhaps future versions will open the referenced tab